mu/interface.py

```diff
@@ -653,7 +653,7 @@ def connect_zoom(self, widget):
     @property
     def current_tab(self):
         """
-        Returns the currently focused tab.
+        Returns the currently focussed tab.
```

Comment:
My IDE flagged it, so I fixed it. When I googled, it seemed valid.  Since I had doubts I pulled it.